website/pages/en/querying/querying-from-an-application.mdx

```diff
@@ -2,49 +2,61 @@
 title: Querying from an Application
 ---
 
-Once a subgraph is deployed to Subgraph Studio or to Graph Explorer, you will be given the endpoint for your GraphQL API that should look something like this:
+Learn how to query The Graph from your application.
 
-**Subgraph Studio (testing endpoint)**
+## Getting GraphQL Endpoint
+
+Once a subgraph is deployed to [Subgraph Studio](https://thegraph.com/studio/) or [Graph Explorer](https://thegraph.com/explorer), you will be given the endpoint for your GraphQL API.
+
+### Subgraph Studio
+
+Your GraphQL API in Studio should look something like this:
 
 ```sh
 Queries (HTTP)
 https://api.studio.thegraph.com/query/<ID>/<SUBGRAPH_NAME>/<VERSION>
 ```
```

Comment:
Not something you changed, but this is not a `sh` code block. I'm also not sure why why need "Queries (HTTP)" at all. I would change it to:
```suggestion
```
https://api.studio.thegraph.com/query/<ID>/<SUBGRAPH_NAME>/<VERSION>
```
```